### PR TITLE
Milestone-4

### PR DESCRIPTION
**Pull request by AkintoyeOlamide**
Adding the logic for the calculator

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      result = Big(num1.times(num2));
+    }
+
+    if (operation === '÷') {
+      result = Big(num1.div(num2));
+    }
+
+    if (operation === '%') {
+      result = Big(num1.div(100));
+    }
```
Comment (Elbie-em) [inline src/logic/operate.js:27]: - [ ] You are creating a new instance BIg in every case after you  have wrapped numberOne and numberTwo. You can simply find the result by e.g
```
if (operation === '+') {
      result = num1.plus(num2);
 }
```

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    if (operation === 'X') {
+      result = Big(num1.times(num2));
+    }
+
+    if (operation === '÷') {
+      result = Big(num1.div(num2));
+    }
+
+    if (operation === '%') {
+      result = Big(num1.div(100));
```
Comment (Elbie-em) [inline src/logic/operate.js]: - [ ] You should use the modifier function for this case here as required.

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡  ♻️ 

Hi @AkintoyeOlamide, you have done a great job so far. I like how you are wrapping your operations in a try-catch block. However, there are some minor changes needed before moving forward. Check my comments below for more details ⬇️ 

*All the best and keep up the good work* 💪 

#### *After applying the requested changes, you can submit another code review request*

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  try {
+    const num1 = Big(+numberOne);
+    const num2 = Big(+numberTwo);
+
```
Comment (uche-inyama) [inline src/logic/operate.js:8]: Why did you prefix the '+' sign in **numberOne & numberTwo**

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi Olamide :slightly_smiling_face: 

Go ahead with the next milestone.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**